tests/integration/cli/projects/geth/ape-config.yaml

```diff
@@ -11,6 +11,8 @@ geth:
       uri: http://localhost:5000
     local:
       uri: http://127.0.0.1:5550  # NOTE: Has to be same as tests/conftest.py::GETH_URI
+      extra_funded_accounts:
+        - 0x999999cf1046e68e36E1aA2E0E07105eDDD1f08E
```

Comment:
Is this a special address for you?

Reply:
It isn't but let me change it to `test_account[99]` if you were to generate that far, for better practice